mount/mountinfo_linux.go

```diff
@@ -21,6 +21,7 @@ package mount
 import (
 	"bufio"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
could we move it to next package group?

Reply:
oh! yes, looks like my editor inserted it